### ISSUE TITLE
Possible lint/uniformity issue with trailing commas.

### ISSUE THREAD

**Issue by ksbrar** [microsoft/FluidFramework#6070]
## Describe the bug
Noticed by @vladsud in https://github.com/microsoft/FluidFramework/pull/5962#discussion_r624246579. I haven't looked into it yet, but potentially we are not consistently enforcing trailing commas?

**Comment by tylerbutler** [comment]
Fwiw, this the current config for the comma dangle rule:

```
        "@typescript-eslint/comma-dangle": [
            "error",
            {
                "arrays": "always-multiline",
                "enums": "always-multiline",
                "exports": "always-multiline",
                "functions": "always-multiline",
                "generics": "never",
                "imports": "always-multiline",
                "objects": "always-multiline",
                "tuples": "always-multiline",
            }
        ],
```

**Comment by vladsud** [comment]
Moving it to unassigned, to keep some number of "good first issue" candidates around :)

**Comment by tylerbutler** [comment]
Fixed in #10087.
